src/Ruleset.php

```diff
@@ -238,7 +238,7 @@ public function __construct(Config $config)
     public function explain()
     {
         $sniffs = array_keys($this->sniffCodes);
-        sort($sniffs);
+        sort($sniffs, (SORT_NATURAL | SORT_FLAG_CASE));
```

Comment:
Would using `natcasesort()` be more legible here?

Reply:
Just checked, yes, the function is available, but is still not the right choice.

`natcasesort()` maintains key association. `sort()` does not.
The `foreach()` loop used in the function uses the (numeric) key to determine whether the end of the array has been used, so the array not being re-indexed would cause it to stop the loop too early or even to display sniffs with the wrong standard.